curseforge/export.go

```diff
@@ -19,9 +20,10 @@ var exportCmd = &cobra.Command{
 	Short: "Export the current modpack into a .zip for curseforge",
 	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		// cobra/viper ensures this value is set if the flag is used. Maybe remove the len Check?
```

Comment:
Yeah you can remove the len check here as well - the latter checks already handle the case where the string is empty, and it's very unlikely that a user would manage to set the flag to `""`